.github/workflows/cross.yml

```diff
@@ -37,20 +37,16 @@ jobs:
           - x86_64-unknown-linux-musl
           # - x86_64-unknown-netbsd  # error in tests "error: test failed, to rerun pass '--lib'", disabled for now
     steps:
-      - name: Checkout Crate
-        uses: actions/checkout@v2
+      - name: "Checkout repo"
```

Comment:
IME quotes are unusual in GH actions ant thus I think they're harder to read.

Reply:
Do you want me to remove all quotes from all of our GitHub actions files?

Reply:
I think that every string on YAML should be quoted.
@Kixunil don't know if you already seen this but here are a bunch of good arguments: https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell

(PS: I wish we could do GH Actions using TOML, I honestly think YAML is very very easy to shoot yourself in the foot...)

Reply:
I knew YAML is bad but I didn't know it's *that* bad. I'd make an exception for the `name:` field since it just contains a human-readable name but if you want to use some kind of linter and ban it entirely, it's fine.